roadmap/implementers-guide/src/runtime/disputes.md

```diff
@@ -123,4 +123,17 @@ applied:
    as a measure not to import one sided disputes. A dispute is raised only if there are two opposing votes so if the
    client is not sending them the dispute is a potential spam.
 4. Remove unconfirmed disputes - if a dispute contains less votes than the byzantine threshold it is removed. This is
-   also a spam precaution. A legitimate client will send only confirmed disputes to the runtime.
\ No newline at end of file
+   also a spam precaution. A legitimate client will send only confirmed disputes to the runtime.
+
+# Rewards and slashing
+
+After the disputes are filtered the validators participating in the disputes are rewarded and more importantly the
+offenders are slashed. Generally there can be two types of punishments:
+* "against valid" - the offender claimed that a valid candidate is invalid.
+* "for invalid" - the offender claimed that an invalid candidate is valid.
+
+Validators participating in inconclusive disputes are not slashed. Thanks to the applied filtering (described in the
```

Comment:
nit: I would also detail what `inconclusive` means in this case .

Reply:
Yes, this is a good suiggestion.